Custom-Video-Driver/Lets-Build-OTPublisher/ExampleVideoCapture.swift

```diff
@@ -58,6 +58,15 @@ class ExampleVideoCapture: NSObject, OTVideoCapture {
     
     var delegate: FrameCapturerMetadataDelegate?
     
+    var cameraPosition: AVCaptureDevice.Position {
+        get {
```

Comment:
I think you could use the `??` operator here.

Something like this:
`return videoInput.device.position ?? .unspecified`

Reply:
Done